pilot/pkg/xds/xdsgen.go

```diff
@@ -60,11 +60,15 @@ func init() {
 	controlPlane = &corev3.ControlPlane{Identifier: string(byVersion)}
 }
 
-func (s *DiscoveryServer) findGenerator(typeURL string, con *Connection) model.XdsResourceGenerator {
+func (s *DiscoveryServer) findGenerator(typeURL string, con *Connection, delta bool) model.XdsResourceGenerator {
```

Comment:
It does not right changing signature of this method. The better way would be to append "delta" to typeURl when it is invoked from delta.go

Reply:
yeah, we could do that